pkg/plugins/policies/meshtcproute/plugin/v1alpha1/plugin_test.go

```diff
@@ -705,32 +705,38 @@ var _ = Describe("MeshTCPRoute", func() {
 
 			rules := core_rules.Rule{
 				Subset: core_rules.MeshSubset(),
-				Conf: api.RuleConf{
-					BackendRefs: []common_api.BackendRef{{
-						TargetRef: builders.TargetRefService("backend"),
-						Weight:    pointer.To(uint(100)),
-					}},
+				Conf: api.Rule{
```

Comment:
can we do something about such mistakes at compile time? can we have some static analysis tool to check that we're casting `Conf` to something that always `Default`?

Reply:
I thought about a few things (like an interface to implement etc). But I couldn't find a good idea I'm open to suggestions if you have a good idea!

Reply:
Maybe it must have a `Default` entry...

Reply:
Yes, just a marker trait with a method like "IsPolicyConf" is probably enough, that it won't end up being implemented twice